internal/archiver/archiver.go

```diff
@@ -78,10 +78,19 @@ type Archiver struct {
 	// WithAtime configures if the access time for files and directories should
 	// be saved. Enabling it may result in much metadata, so it's off by
 	// default.
-	WithAtime   bool
-	IgnoreInode bool
+	WithAtime bool
+
+	ChangeIgnore ChangeIgnore
 }
 
+// Change detection flags. See doc/040_backup.rst for details.
+type ChangeIgnore = uint8
```

Comment:
What about adding methods `Set(..[]iunt8)` and `IsSet(uint8) bool` on this bitset type such that 
```
arch.ChangeIgnore.Set(ChangeIgnoreCtime, ChangeIgnoreInode)

ignoreCtime := ignore.IsSet(ChangeIgnoreCtime)
```
can be used?

Or maybe an interface for doing somethin like
```
arch.ChangeIgnore.SetIgnoreCtime()
arch.ChangeIgnore.SetIgnoreInode()

ignoreCtime := ignore.IsIgnoreCtime()
```

 Would IMO improve readability.

Reply:
I prefer a straight bit field: & and |= have fixed semantics and the get and set are in two places. With getters and setters, you have to check three areas of the code to ensure that it does what you think it does.